ivy/functional/frontends/numpy/indexing_routines/indexing_like_operations.py

```diff
@@ -19,3 +21,9 @@ def diag_indices(n, ndim=2):
 @to_ivy_arrays_and_back
 def diag(v, k=0):
     return ivy.diag(v, k=k)
+
+
+@to_ivy_arrays_and_back
+def unravel_index(indices, shape):
```

Comment:
Can you add, `order='C'` as argument. Have to make this function exactly same as numpy function. (You can leave this argument unused)